tests/test_main.py

```diff
@@ -0,0 +1,64 @@
+"""
+Test for main page using fastapi test client.
+"""
+from datetime import datetime
+from fastapi.testclient import TestClient
+from main import app
+import os
+import sys
+import random
+import main
+from messages import reminders
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+# Declaring test client
+client = TestClient(app)
+
+# Test case for index page
+def test_index():
+    response = client.get("/")
+    assert response.status_code == 200
+    assert response.json() == {"msg":"This is the endpoint for the home page. /message \
+        and /reminder are more useful starting points."}
+
+# Test for status code for get message
+def test_get_message():
+    response = client.get("/message")
+    assert response.status_code == 200
+
+# Test for asserting random message is in the file
+def test_get_message_text():
+    "asserting random message is in the file"
+    response = client.get("/message")
+    message = response.json()
+    with open(main.CULTURE_FILE, 'r') as file_handler:
+        lines = [line.strip() for line in file_handler]
+    assert message['msg'] in lines
+
+# Test for Reminders status code
+def test_get_reminder():
+    response = client.get("/reminder")
+    assert response.status_code == 200
+
+# Test for asserting correct reminder sent
+def test_get_reminder_text_check():
+    response = client.get("/reminder")
+    message = response.json()
+    weekday = main.get_weekday()
+    lines = reminders.messages.get(weekday, [''])
+    message = message.get('msg', '')
+    message = message.split('Reminder:</b>')[-1].lstrip()
+    assert message in lines
+
+# Test for sep20_interns status code
+def test_get_sep20_message():
+    response = client.get("/sep20-interns")
+    assert response.status_code == 200
+
+# Test for asserting correct message for sep20_interns
+def test_get_sep20_message_text_check():
+    response = client.get("/sep20-interns")
+    message = response.json()
+    with open(main.SEP20_INTERNS_FILE, 'r') as file_handler:
+        lines = [line.strip() for line in file_handler]
+    assert message['msg'] in lines
```

Comment:
I believe pylint likes one empty line at the end of this file. Can you please run pylint for this?